packages/example-generate-sm/src/index.ts

```diff
@@ -1,15 +1,21 @@
 import { ContractGenerator } from '@aloxide/abstraction';
-import { EOSContractAdapter } from '@aloxide/bridge';
+import { EOSContractAdapter, ICONContractAdapter, ModelContractAdapter } from '@aloxide/bridge';
 import { createLogger } from 'bunyan';
 import path from 'path';
 
+const resultPath = path.resolve(__dirname, '../out');
```

Comment:
`outputPath` is more appropriate name, as it's also consistent with line 15.

Reply:
fixed